planning/behavior_velocity_planner/autoware_behavior_velocity_roundabout_module/config/roundabout.param.yaml

```diff
@@ -0,0 +1,41 @@
+/**:
+  ros__parameters:
+    roundabout:
+      common:
+        attention_area_length: 200.0
```

Comment:
Can you remove this parameter from the config and README ?
https://autowarefoundation.github.io/autoware_universe/pr-11142/planning/behavior_velocity_planner/autoware_behavior_velocity_roundabout_module/
As attention_area seems to be equal to the set of all lanelets inside the roundabout